RELEASE.md

```diff
@@ -0,0 +1,128 @@
+# Delivery workflow
+
+We use 9 swim lanes on the [Jira][jira] board.
+
+**1. Sprint Backlog**
+
+- meets [definition of ready](#definition-of-ready)
+
+**2. Development**
+
+- active local development
+- push to remote (branch must contain ticket number `ER-123`)
+- mark PR as `draft` and summarise deliverable
+- associate to release candidate milestone
+
+**3. Blocked**
+
+- stalled or parked
+- label PR as `blocked`
+
+**4. Developer Peer Review**
+
+- `content` deployment
+- `ER-456` branch
+- <https://ey-recovery-pr-123.london.cloudapps.digital>
+- [feature deployments][deployments]
+- meets [definition of done](#definition-of-done)
+- mark PR as `ready` and [request review](#review-process)
+- once approved move ticket to `#5`
+
+**5. Quality Assurance**
+
+- `content` deployment
+- seek business analyst sign-off
+- confirm the acceptance criteria
+- label PR as `pass` or `fail`
+- once passed merge to `main` branch and move ticket to `#6`
+
+**6. Integration**
+
+- `development` deployment
+- `main` branch
+- <https://ey-recovery-dev.london.cloudapps.digital>
+- update accessibility and quality checks as required
+
+**7. Approval**
+
+- `staging` deployment
+- `rcx.x.x` release candidate tag
+- <https://ey-recovery-staging.london.cloudapps.digital>
+- [open milestones (release candidates)][release-candidates]
+- seek product owner sign-off
+
+**8. Release**
+
+- `production` deployment
+- `vx.x.x` version tag
+- <https://ey-recovery.london.cloudapps.digital>
+- [close milestone][released-versions] and rename from `rc` to `v`
+- periodically publish a [release][releases] CHANGELOG
+
+**9. Post Release**
+
+- post release tasks
+
+
+## Definition of Ready
+
+- story is estimated
+- acceptance criteria defined
```

Comment:
we don't actually estimate stories - more like, the technical aspects are understood by the team well enough to know it is doable within a sprint

Reply:
Shamila wants to change this